src/commonMain/kotlin/com/algolia/search/model/response/ResponseSearch.kt

```diff
@@ -211,7 +212,8 @@ public data class ResponseSearch(
     /**
      * Meta-information as to how the query was processed.
      */
-    @SerialName(KeyExplain) val explainOrNull: Explain? = null
+    @SerialName(KeyExplain) val explainOrNull: Explain? = null,
+    @SerialName(KeyAppliedRules) val appliedRulesOrNull: List<JsonObject>? = null
```

Comment:
Instead of using `JsonObject`, could we use a strong type? The [API doc](https://www.algolia.com/doc/rest-api/logs/#method-param-appliedrules) states that it will be `objectID` to `String` If I am not mistaken. (Also shown at the bottom of this [Helpscout ticket](https://secure.helpscout.net/conversation/1157336776/253277/))

Reply:
after discussion offline, we'll keep JsonObject (same was done in Java client)